test/unit-tests/utilities/version.test.ts

```diff
@@ -12,103 +12,103 @@
-        assert.equal(new Version(5, 10, 1).isLessThanOrEqual(new Version(5, 10, 0)), false);
-        assert.equal(new Version(6, 0, 0).isLessThanOrEqual(new Version(5, 10, 1)), false);
+        expect(new Version(5, 10, 1).isLessThanOrEqual(new Version(6, 0, 0))).to.equal(true);
+        expect(new Version(5, 9, 0).isLessThanOrEqual(new Version(5, 10, 0))).to.equal(true);
+        expect(new Version(5, 10, 0).isLessThanOrEqual(new Version(5, 10, 1))).to.equal(true);
+        expect(new Version(5, 10, 1).isLessThanOrEqual(new Version(5, 10, 1))).to.equal(true);
+        expect(new Version(5, 10, 0).isLessThanOrEqual(new Version(5, 9, 0))).to.equal(false);
+        expect(new Version(5, 10, 1).isLessThanOrEqual(new Version(5, 10, 0))).to.equal(false);
+        expect(new Version(6, 0, 0).isLessThanOrEqual(new Version(5, 10, 1))).to.equal(false);
     });
 
     test("isGreaterThan", () => {
-        assert.equal(new Version(5, 10, 1).isGreaterThan(new Version(6, 0, 0)), false);
-        assert.equal(new Version(5, 9, 0).isGreaterThan(new Version(5, 10, 0)), false);
-        assert.equal(new Version(5, 10, 0).isGreaterThan(new Version(5, 10, 1)), false);
-        assert.equal(new Version(5, 10, 1).isGreaterThan(new Version(5, 10, 1)), false);
-        assert.equal(new Version(5, 10, 0).isGreaterThan(new Version(5, 9, 0)), true);
-        assert.equal(new Version(5, 10, 1).isGreaterThan(new Version(5, 10, 0)), true);
-        assert.equal(new Version(6, 0, 0).isGreaterThan(new Version(5, 10, 1)), true);
+        expect(new Version(5, 10, 1).isGreaterThan(new Version(6, 0, 0))).to.equal(false);
+        expect(new Version(5, 9, 0).isGreaterThan(new Version(5, 10, 0))).to.equal(false);
+        expect(new Version(5, 10, 0).isGreaterThan(new Version(5, 10, 1))).to.equal(false);
+        expect(new Version(5, 10, 1).isGreaterThan(new Version(5, 10, 1))).to.equal(false);
+        expect(new Version(5, 10, 0).isGreaterThan(new Version(5, 9, 0))).to.equal(true);
+        expect(new Version(5, 10, 1).isGreaterThan(new Version(5, 10, 0))).to.equal(true);
+        expect(new Version(6, 0, 0).isGreaterThan(new Version(5, 10, 1))).to.equal(true);
     });
 
     test("isGreaterThanOrEqual", () => {
-        assert.equal(new Version(5, 10, 1).isGreaterThanOrEqual(new Version(6, 0, 0)), false);
-        assert.equal(new Version(5, 9, 0).isGreaterThanOrEqual(new Version(5, 10, 0)), false);
-        assert.equal(new Version(5, 10, 0).isGreaterThanOrEqual(new Version(5, 10, 1)), false);
-        assert.equal(new Version(5, 10, 1).isGreaterThanOrEqual(new Version(5, 10, 1)), true);
-        assert.equal(new Version(5, 10, 0).isGreaterThanOrEqual(new Version(5, 9, 0)), true);
-        assert.equal(new Version(5, 10, 1).isGreaterThanOrEqual(new Version(5, 10, 0)), true);
-        assert.equal(new Version(6, 0, 0).isGreaterThanOrEqual(new Version(5, 10, 1)), true);
+        expect(new Version(5, 10, 1).isGreaterThanOrEqual(new Version(6, 0, 0))).to.equal(false);
```

Comment:
I'd argue to.be.false or true or undefined read more naturally to make clear why chai makes things clearer

Reply:
Good point. Done.